content/docs/pipelines/tutorials/pipelines-tutorial.md

```diff
@@ -158,7 +146,7 @@ Deploy Kubeflow on GCP:
 1. (Optional) If you want to examine your cluster while waiting for the UI to
    be available, you can use `kubectl` to connect to your cluster:
 
-  * Connect your local `kubectl` session to the cluster:
+  * Connect your Cloud Shell session to the cluster:
```

Comment:
nit: it might make sense to still reference kubectl here (maybe "Connect your Cloud Shell kubectl session"?). Users may not understand how this connection works